python/sglang/srt/layers/attention/vision.py

```diff
@@ -620,6 +624,8 @@ def __init__(
             tp_size=self.tp_size,
             prefix=add_prefix("proj", prefix),
         )
+        self.aux_stream = torch.cuda.Stream()
```

Comment:
Will this cause a new stream to be created for each layer?

Reply:
It refers to the implementation in trtllm, for example the llama model, it is implemented likewise.

Reply:
Otherwise we need to pass the self.aux_stream along in the whole path, which is a non-trivial refactor.